config/default/core.entity_view_display.node.localgov_news_article.default.yml

```diff
@@ -37,6 +37,11 @@ content:
     third_party_settings: {  }
     weight: 4
     region: content
+  content_moderation_control:
+    settings: {  }
+    third_party_settings: {  }
+    weight: 0
+    region: content
```

Comment:
This shouldn't be there. We've been struggling to keep these from returning for months.